cmd/buildah/mount.go

```diff
@@ -41,21 +42,20 @@ func mountCmd(c *cli.Context) error {
 	}
 
 	if len(args) == 1 {
-		name = args[0]
-
+		name := args[0]
```

Comment:
Need to either revert this line, or remove the declaration of `name` at the start of `main()`.